lib/pymedphys/_gamma/implementation/shell.py

```diff
@@ -97,8 +98,7 @@ def gamma_shell(
         The number of bytes of RAM available for use by this function. Defaults
         to 0.8 times your total RAM as determined by psutil.
     quiet : bool, optional
-        Used to quiet informational printing during function usage. Defaults to
-        False.
+        (Deprecated but maintained for now for backwards compatibility)
```

Comment:
Maybe a note around what logging parameters can be passed to achieve the desired verbosity?

Reply:
I've now added something.